bdk_esplora/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "bdk_esplora"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+bdk_chain = { path = "../bdk_chain", features = ["serde", "miniscript", "file_store"] }
+esplora-client = { git = "https://github.com/rajarshimaitra/rust-esplora-client.git", branch = "get_recent_blocks"}
```

Comment:
this must depend on a crates.io release. We have everything we need in the latest release (method names have probably changed).